<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ic_logo.png</normaloff>images/ic_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Employee Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Employee Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmployeeName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmployeeEmail">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Man">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Man</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Woman">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Woman</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_add.png</normaloff>images/ic_add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_save.png</normaloff>images/ic_save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_save.png</normaloff>images/ic_save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_close.png</normaloff>images/ic_close.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetEmployee">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>581</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
